### PR TITLE
Add links to promote reviews by non core-developers

### LINKED ISSUE #194: Promote helping with reviews more
This might fall out as part of #189 , but as of right now all of the highly promoted activities for people to contribute by are (in order of the index):

1. Docs
2. Test coverage
3. Silence warnings
4. Fixing issues from buildbots
5. Fixing "easy" issues
6. Triaging issues

There is no mention of helping to review PRs. And idea 6 for helping triage issues is at the end. Unfortunately triaging issues and helping to review PRs can be the most helpful as they help *reduce* the workload for core devs versus ideas 1 - 5 which *make* work for core devs.

Now it might not entirely make sense to have people start with triage and code review, but the point should be made somehow that activities that reduce workload is **really** important.

### PR DESCRIPTION
**Pull request by mariocj89**
Add a link to the open PR with tag [`awaiting-review`](https://github.com/python/cpython/pulls?q=is%3Apr+is%3Aopen+label%3A%22awaiting+review%22) and another one pointing to [how to review pull requests](https://docs.python.org/devguide/pullrequest.html#how-to-review-a-pull-request).

The objective is to try to promote non-core developers to help out with the reviews as stated in #194

Closes #194

### CODE REVIEW

**Comment by brettcannon** [comment]
Should we add https://github.com/python/cpython/pulls?q=is%3Apr+is%3Aopen+label%3A%22awaiting+review%22 as a link to help people find PRs that are floundering with no reviews?

**Comment by mariocj89** [comment]
@brettcannon the link is already present. It is the main objective of this PR, it has been added to index.rst

**Comment by brettcannon** [comment]
Ah, sorry for overlooking that.

File: index.rst [hunk 1]
```diff
@@ -189,6 +189,8 @@ Guide for contributing to Python:
 * Advanced tasks for once you are comfortable
     * :doc:`silencewarnings`
     * Fixing issues found by the :doc:`buildbots <buildbots>`
+    * Helping out with `Open Reviews`_.
```
Comment (Mariatta) [inline index.rst]: "Open Reviews" might not immediately make sense to people. Perhaps say 'Helping out with reviewing open pull requests'?

File: index.rst [hunk 2]
```diff
@@ -354,3 +356,4 @@ Full Table of Contents
 .. _IronPython: http://ironpython.net/
 .. _Stackless: http://www.stackless.com/
 .. _Issue tracker: https://bugs.python.org/
+.. _Open Reviews: https://github.com/python/cpython/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3A%22awaiting%20review%22
```
Comment (Mariatta) [inline index.rst]: This should now be changed, from `.. _Open Reviews: ` to `.. _open pull requests:`.

**Comment by mariocj89** [comment]
ouch, should be fixed now. @Mariatta should we say "Helping out with reviewing pull requests" or "Helping out reviewing pull requests".

**Comment by Mariatta** [comment]
This looks good. Thanks @mariocj89 and @willingc :taco:

**Pull request merged**